vector/src/main/AndroidManifest.xml

```diff
@@ -364,7 +439,9 @@
         A media button receiver receives and helps translate hardware media playback buttons,
         such as those found on wired and wireless headsets, into the appropriate callbacks in your app.
            -->
-        <receiver android:name="androidx.media.session.MediaButtonReceiver">
+        <receiver
+            android:name="androidx.media.session.MediaButtonReceiver"
+            android:exported="true" >
```

Comment:
Maybe this will also work with `exported="false"` ?

Reply:
Can you test it please?

Reply:
I tested with wireless earbuds if click on call events work with `exported="false"` and they seem to work (**onMediaButtonEvent** function is triggered). So I will update it with **false**. If there is an other usage other than that, that you can think of let me know

Reply:
thanks!